File: SeleniumJava/com.crm.seleniumM4/src/locators/LocatingByID.py
from selenium import webdriver
import time
from selenium.webdriver.common.by import By

driver = webdriver.Chrome(); 
driver.maximize_window(); 
url = "https://demowebshop.tricentis.com/";
driver.get(url)
fetched_url = driver.current_url
if(url == fetched_url) :
    print("I am on dws website")
    time.sleep(2)
    
    # clicking on register button 
    driver.find_element(By.ID, "small-searchterms").send_keys("iPhone")
    search = find_element(By.ID, )
    driver.find_element(By.ID, )
    time.sleep(5)
    driver.close()
else:
    print("wrong website")
    driver.close()

File: SeleniumJava/com.crm.seleniumM4/src/locators/LocatingByID.java
package locators;

import org.openqa.selenium.By;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class LocatingByID {

	public static void main(String[] args) throws InterruptedException {
		
		ChromeDriver driver = new ChromeDriver(); 
		String url = "https://demowebshop.tricentis.com/";
		
		driver.manage().window().maximize();
		driver.get(url);
		
		if(driver.getCurrentUrl().equals(url)) {
			System.out.println("I am on DWS Website");
			
			Thread.sleep(2000);
			
			driver
			//.findElement(By.className("search-box-text ui-autocomplete-input"))
			// don't use it if there are spaces present in className. 
			// it is showing multiple class names. 
			.findElement(By.id("small-searchterms"))
			.sendKeys("iPhone");
			WebElement search = driver.findElement(By.id("small-searchterms"));
			Thread.sleep(3000);
			search.sendKeys(Keys.RETURN);
			
			Thread.sleep(5000);
			driver.close();
			
		}
		else {
			System.out.println("I am not on dws websiite");
			driver.close();
		}
		
		driver.close();

	}

}
